frontend/src/components/app-status/index.vue

```diff
@@ -243,6 +243,6 @@ defineExpose({
     border: 1px solid var(--el-color-warning);
     background-color: transparent;
     padding: 8px 8px;
-    width: 70px;
+    width: 78px;
 }
 </style>
```

Comment:
The change from `width: 70px;` to `width: 78px;` does not introduce any immediate irregularities, issues, or optimizations. This is a minor adjustment that affects the overall size of the component but has no functional impact.

However, it's worth noting that if this component is used frequently and its appearance significantly impacts user experience, making small adjustments like these could be beneficial for alignment or spacing purposes without causing major visual anomalies.